src/apps/watchfaces/OswAppWatchfaceBinary.cpp

```diff
@@ -82,7 +80,7 @@ void OswAppWatchfaceBinary::loop() {
     if(hal->btnHasGoneDown(BUTTON_2)) {
         hal->decreaseBrightness(25);
     }
-    drawWatch(hal->getCanvas()->getGraphics2D());
+    drawWatch(hal->getCanvas());
```

Comment:
Remove parameter?

Reply:
I guess, didn't I remove that function entirely? It still compiles?!

Reply:
I remove it. Is it ok